CRM/Event/Form/Registration/AdditionalParticipant.php

```diff
@@ -180,32 +180,6 @@ public function buildQuickForm() {
     if ($this->_values['event']['is_monetary']) {
       CRM_Event_Form_Registration_Register::buildAmount($this);
     }
-    $first_name = $last_name = NULL;
-    $pre = $post = [];
-    foreach ([
-      'pre',
-      'post',
-    ] as $keys) {
-      if (isset($this->_values['additional_custom_' . $keys . '_id'])) {
-        $this->buildCustom($this->_values['additional_custom_' . $keys . '_id'], 'additionalCustom' . ucfirst($keys));
```

Comment:
seems like this is required to load profiles for additional participants? Currently, no fields are displayed on the page.

![image](https://user-images.githubusercontent.com/5929648/68024254-903f2d00-fccf-11e9-9c89-b0700b27b208.png)

Reply:
Have requested a PR here https://github.com/civicrm/civicrm-core/pull/15698. Seems like only this line is required ?